cmd/prepare-storage.go

```diff
@@ -196,9 +196,16 @@ func retryFormattingDisks(firstDisk bool, firstEndpoint string, storageDisks []S
 	defer close(doneCh)
 
 	// Wait on the jitter retry loop.
-	for range newRetryTimer(time.Second, time.Second*30, MaxJitter, doneCh) {
+	for retryCounter := range newRetryTimer(time.Second, time.Second*30, MaxJitter, doneCh) {
 		// Attempt to load all `format.json`.
 		formatConfigs, sErrs := loadAllFormats(storageDisks)
+		if retryCounter > 5 {
+			for i, e := range sErrs {
```

Comment:
> What about the other errors?

Here, we delay showing some errors that could disappear later like disconnected node in our case. Other errors could be shown in other PRs